samples/cpp/tutorial_code/ImgProc/HitMiss.cpp

```diff
@@ -15,18 +15,28 @@ int main(){
         0, 255, 0, 255, 0, 0, 255, 0,
         0, 255, 255, 255, 0, 0, 0, 0);
 
-    Mat kernel = (Mat_<uchar>(3, 3) <<
+    Mat kernel = (Mat_<int>(3, 3) <<
         0, 1, 0,
         1, -1, 1,
         0, 1, 0);
 
     Mat output_image;
     morphologyEx(input_image, output_image, MORPH_HITMISS, kernel);
 
-    namedWindow("Original", CV_WINDOW_NORMAL);
+    const int rate = 10;
+    
```

Comment:
@logic1988 You need to remove the trailing whitespaces of lines 27, 32, 35, 38, and 40 for the build to be successful (http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/8289/steps/whitespace%20opencv/logs/stdio)